applications/ContactStructuralMechanicsApplication/python_scripts/alm_contact_process.py

```diff
@@ -68,38 +67,45 @@ def __init__(self, model_part, settings):
             "model_part_name"             : "Structure",
             "computing_model_part_name"   : "computing_domain",
             "contact_model_part"          : "Contact_Part",
-            "axisymmetric"                : false,
             "assume_master_slave"         : "Parts_Parts_Auto1",
             "contact_type"                : "Frictionless",
-            "frictional_law"              : "Coulomb",
-            "search_factor"               : 3.5,
-            "active_check_factor"         : 0.01,
-            "max_number_results"          : 1000,
-            "bucket_size"                 : 4,
+            "interval"                    : [0.0,"End"],
             "normal_variation"            : "NO_DERIVATIVES_COMPUTATION",
-            "manual_ALM"                  : false,
-            "stiffness_factor"            : 1.0,
-            "penalty_scale_factor"        : 1.0,
-            "use_scale_factor"            : true,
-            "penalty"                     : 0.0,
-            "scale_factor"                : 1.0e0,
+            "frictional_law"              : "Coulomb",
             "tangent_factor"              : 0.1,
-            "type_search"                 : "InRadius",
-            "check_gap"                   : "CheckMapping",
-            "database_step_update"        : 1,
             "integration_order"           : 2,
-            "adapt_penalty"               : false,
-            "max_gap_factor"              : 1.0e-3,
-            "dynamic_search"              : false,
-            "double_formulation"          : false,
-            "debug_mode"                  : false,
-            "remeshing_with_contact_bc"   : false
+            "remeshing_with_contact_bc"   : false,
+            "search_parameters" : {
+                "type_search"                 : "InRadius",
```

Comment:
style guide...

Reply:
Ok

Reply:
there are several ones, just sayin, I am not a blocker on this